lib/view_component/slotable_v2.rb

```diff
@@ -174,11 +174,23 @@ def register_slot(slot_name, collection:, callable:)
         self.registered_slots[slot_name] = slot
       end
 
-      def validate_slot_name(slot_name)
+      def validate_plural_slot_name(slot_name)
```

Comment:
We could use inflections here to make sure the plural and singular versions of the slot name don't clash with things in future, but until we're confident of that, I thought I'd keep things simple for now.